applications/rogue/main.tf

```diff
@@ -1,17 +1,26 @@
 # This template builds a Rogue application instance.
 #
-# Manual setup steps:
-#   - Set 'APP_KEY' by running 'php artisan generate:key'.
-#   - Set 'ROGUE_API_KEY' to a random string for this instance's v2 API key.
-#   - Create app-specific user & machine OAuth clients (via Aurora) and set the
-#     values for 'NORTHSTAR_URL', 'NORTHSTAR_AUTH_ID', 'NORTHSTAR_AUTH_SECRET',
-#      'NORTHSTAR_CLIENT_ID', and 'NORTHSTAR_CLIENT_SECRET'
-#   - Set 'BLINK_URL', 'BLINK_USERNAME', and 'BLINK_PASSWORD' if using
-#     Blink, and set 'DS_ENABLE_BLINK' environment variable to 'true'.
-#   - Set 'DS_ENABLE_BLINK', 'DS_ENABLE_GLIDE', 'DS_ENABLE_PUSH_TO_QUASAR',
-#     'DS_ENABLE_V3_QUANTITY_SUPPORT', 'FASTLY_API_TOKEN', 'FASTLY_SERVICE_ID',
-#     'FASTLY_URL', 'SLACK_ENDPOINT', and 'SLACK_WEBHOOK_INTEGRATION_URL'.
-#   - Finally, set '/newrelic/api-key' in SSM to the New Relic API Key, if using.
+# To create a new Rogue instance, 
+#   1. Configure a new 'rogue' module with the required variable arguments & apply.
+#   2. The initial apply will fail trying to create 'heroku_formation' resources
+#      because code hasn't been deployed. Deploy this application's `master` branch
+#      via Heroku's web interface & then re-run the apply. It should be successful.
+#   3. Generate an 'APP_KEY' by running 'php artisan generate:key' on a local instance of
+#      this application. Copy-paste that value into `APP_KEY` in Heroku's web interface.
```

Comment:
If the `TEST_APP_KEY` in this PR looks good, we'll be able to remove this manual step!